core/src/plugin/process_info.rs

```diff
@@ -50,18 +50,9 @@ pub struct ProcBuffers {
 }
 
 impl ProcBuffers {
-    pub fn audio_inputs_silent(&self, use_slow_check: bool, frames: usize) -> bool {
+    pub fn audio_inputs_silent(&self, frames: usize) -> bool {
         for buf in self.audio_in.iter() {
-            if !buf.is_silent(use_slow_check, frames) {
-                return false;
-            }
-        }
-        true
-    }
```

Comment:
Again, I want plugins to be able to use either a quick check with the constant masks, or a thorough check.